src/org/opendatakit/briefcase/ui/export/AggregateServerConnectionConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.opendatakit.briefcase.ui.export;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class AggregateServerConnectionConfiguration {
+  private URL url;
+  private String username;
+  private char[] password;
+
+  public AggregateServerConnectionConfiguration(String url, String username, char[] password) throws MalformedURLException {
+    this.url = new URL(url);
+    this.username = username;
+    this.password = password;
+  }
+
+  public URL getUrl() {
+    return url;
+  }
+
+  public void setUrl(String url) throws MalformedURLException {
```

Comment:
Don't throw a checked exception. Wrap the `new URL()` inside a try-catch and wrap the `MalformedURLException` inside a `BriefcaseException`

You'll need to change `BriefcaseException` to this: 
```java
public class BriefcaseException extends RuntimeException {
  public BriefcaseException(String message) {
    super(message);
  }

  public BriefcaseException(String message, Throwable cause) {
    super(message, cause);
  }
}
```